haskell/src/Monocle/Api/Server.hs

```diff
@@ -299,6 +299,36 @@ crawlerCommitInfo request = do
         . Enumerated
         $ Right err
 
+-- | /suggestions endpoint
+searchSuggestions :: SearchPB.SuggestionsRequest -> AppM SearchPB.SuggestionsResponse
+searchSuggestions request = do
+  tenants <- getConfig
+  let SearchPB.SuggestionsRequest {..} = request
+
+  let tenantM = Config.lookupTenant tenants (toStrict suggestionsRequestIndex)
+
+  case tenantM of
+    Just tenant -> do
+      now <- getCurrentTime
+      runTenantQueryM tenant (emptyQ now) $ do Q.getSuggestions
+    Nothing ->
+      -- Simply return empty suggestions in case of unknown tenant
+      pure $
+        SearchPB.SuggestionsResponse
+          (V.fromList [])
+          (V.fromList [])
+          (V.fromList [])
+          (V.fromList [])
+          (V.fromList [])
```

Comment:
```suggestion
          mempty mempty mempty mempty mempty
```

Makes me wonder if proto3-suite creates a Monoid instance when the message only has repeated elements. In that case the Nothing case could simply be `mempty`.